app/views/idv/shared/_ssn.html.erb

```diff
@@ -30,11 +30,13 @@ locals:
 
 <% if IdentityConfig.store.proofing_device_profiling_collecting_enabled %>
   <% unless IdentityConfig.store.lexisnexis_threatmetrix_org_id.empty? %>
-    <%= javascript_include_tag "https://h.online-metrix.net/fp/tags.js?org_id=#{IdentityConfig.store.lexisnexis_threatmetrix_org_id}&session_id=#{flow_session[:threatmetrix_session_id]}", nonce: true %>
-    <noscript>
-      <iframe style="width: 100px; height: 100px; border: 0; position: absolute; top: -5000px;" src="https://h.online-metrix.net/fp/tags?org_id=<%= IdentityConfig.store.lexisnexis_threatmetrix_org_id %>&session_id=<%= flow_session[:threatmetrix_session_id] %>">
-      </iframe>
-    </noscript>
+    <% unless flow_session[:threatmetrix_session_id].nil? %>
```

Comment:
I prefer positive checks to `unless`, how about:
```suggestion
    <% if flow_session[:threatmetrix_session_id].present? %>
```

(and since we're here, same for the `unless` above, I'd change it to a `.present?`)